server.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+
+const mongoose = require("mongoose");
+const routes = require("./routes");
+const app = express();
+const PORT = process.env.PORT || 3001;
+
+// Define middleware here
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+// Serve up static assets (usually on heroku)
+if (process.env.NODE_ENV === "production") {
+  app.use(express.static("client/build"));
+}
+// Add routes, both API and view
+app.use(routes);
+
+// Connect to the Mongo DB
+mongoose.connect(process.env.MONGODB_URI || "mongodb://mollyclaire:gracie12@ds353358.mlab.com:53358/heroku_rnr4fs91");
```

Comment:
I found this article interesting https://medium.com/@vsvaibhav2016/best-practice-of-mongoose-connection-with-mongodb-c470608483f0

I haven't worked in-depth with Mongoose, myself. (Only a couple times in a tutorial setting). It looks like most people recommend creating a `database.js` file and then you import that into your `server.js`. In `database.js` you can do some error handling if the database connection fails.

More specifically, if your `process.env.MONGODB_URI` is missing or wrong, you can handle that instead of falling back to a hardcoded URL with credentials.

It's best to avoid putting your credentials in the code itself because anyone can see & abuse that. I noticed you created a connection in `seedDB.js` as well. Instead, you can just import your new `database.js` there as well.